examples/30_extended/tasks_tutorial.py

```diff
@@ -196,11 +196,11 @@
     # Error code for 'task already exists'
     if e.code == 614:
         # Lookup task
-        tasks = openml.tasks.list_tasks(data_id=128, output_format='dataframe').to_numpy()
-        tasks = tasks[tasks[:, 4] == "Supervised Classification"]
-        tasks = tasks[tasks[:, 6] == "10-fold Crossvalidation"]
-        tasks = tasks[tasks[:, 19] == "predictive_accuracy"]
-        task_id = tasks[0][0]
+        tasks = openml.tasks.list_tasks(data_id=128, output_format='dataframe') #.to_numpy()
+        tasks = tasks[tasks.loc[:, "task_type"] == "Supervised Classification"]
+        tasks = tasks[tasks.loc[:, "estimation_procedure"] == "10-fold Crossvalidation"]
+        tasks = tasks[tasks.loc[:, "evaluation_measures"] == "predictive_accuracy"]
+        task_id = tasks.loc[:, "tid"].values[0]
```

Comment:
You can access columns directly as if they were attributes (if the naming follows legal attribute naming). E.g. `tasks.task_type` instead of `tasks.loc[:, "task_type"]`. As I understand this should be identical, but (in my opinion) at lot easier to parse?

Or use panda's query functionality:
```
tasks.query('task_type == "Supervised Classification" '
            'and estimation_procedure == "10-fold Crossvalidation" '
            'and evaluation_measures == "predictive_accuracy"')
```

Reply:
In a recent push I made I received feedback that pandas might do away with the support for named access to a column attribute such as `task.task_type`. Hence, I am using `loc`. 
Though I do agree that direct column access makes the code readable.
Also, I too wanted to use a query but kept the structure that was already there wrt filtering hierarchically. 
I'll make it into a query and push again.

Reply:
I think this could be simplified to `task_id = tasks['tid'][0]`. However, I think we should rather try to get the new version out than doing minor simplifications.